### DISCUSSION TITLE
Does this work with deno.js

### DISCUSSION THREAD

**Discussion by topperspal** [PrismJS/prism#3319] [Q&A]
Does this work with deno.js

**Comment by RunDevelopment** [comment]
Yes, and no.

It doesn't work with Deno out of the box, AFAIK (haven't tried it). We do have a special [`loadLanguages` function for NodeJS](https://prismjs.com/#basic-usage-node), so Deno's Node-compatibility mode might make it work. (In case you try this, `require` is the only NodeJS-specific function `loadLanguages` uses.)

We also have a [babel plugin](https://prismjs.com/#basic-usage-bundlers) that generates ESM imports for Prism. These imports should be compatible with Deno.

If all else fails, you could also download a bundle from our [download page](https://prismjs.com/download.html). These bundles are self-contained and only use ES5 features and APIs. They should work in every JS runtime.
